src/wmnds/components/content-tiles/travel-tips/_travel-tips.scss

```
@@ -2,7 +2,9 @@
   &--purple {
```

```suggestion
 &__block {
```

Change this class name to block so that it makes more sense. 
Also, as it is not a modifier any more and is a block element according to BEM standards it should be '__' not '--'.